index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Valentin Lopan - Junior Frontend Developer</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header id="about">
+        <nav>
+            <ul id="nav" class="nav">
+                <li><a class="smoothscroll" href="#about">About</a></li>
+                <li><a class="smoothscroll" href="#skills">Skills</a></li>
+                <li><a class="smoothscroll" href="#works">Works</a></li>
+                <li><a class="smoothscroll" href="#education">Education</a></li>
+            </ul>
+        </nav>
+        <div class="row">
+            <div class="three columns">
+                <img class="profile-pic" src="images/profile.jpg" alt="" />
+            </div>
+            <div class="nine columns main-col">
+
+                <h2>Hi! I'm Valentin Lopan!</h2>
+
+                <p>
+                    I'm Junior Frontend Developer from Minsk with 3 years experience in IT.
+                    I started my career as a C # automation engineer.
+                    But I always liked to visually see my work, so I retrain to frontend developer.
+                    I want to make convenient and beautiful interfaces that will help people solve their problems
+                    faster and easier.
+                </p>
+
+                <h2>Contact Details</h2>
+                <p class="address">
+                    <span>Valentin Lopan</span><br>
+                    <span>Minsk, Belarus</span><br>
+                    <span>+375(44) 707 69 95</span><br>
+                    <a href="mailto:valiantsin.lopan@gmail.com">valiantsin.lopan@gmail.com</a>
+                </p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="twelve columns">
```

Comment:
I guess what name "twelve", "nine", "three" means, but in this layout you have two sizes = two columns (25%, 75%) only (+ full size div which could be rather "container" than "column").